remotespark/kernels/kernelmagics.py

```diff
@@ -163,6 +190,7 @@ def spark(self, line, cell="", local_ns=None):
     @argument("-n", "--maxrows", type=int, default=None, help="Maximum number of rows that will be pulled back "
                                                                         "from the server for SQL queries")
     @argument("-r", "--samplefraction", type=float, default=None, help="Sample fraction for sampling from SQL queries")
+    @_event
```

Comment:
They make the events table noisy though, since we won't build any metrics from them. I vote to remove `@_event` from the `spark` and `sql` methods.